src/promptflow/tests/sdk_cli_azure_test/e2etests/test_azure_cli_perf.py

```diff
@@ -49,7 +73,7 @@ def operation_scope_args(subscription_id: str, resource_group_name: str, workspa
     "vcr_recording",
 )
 class TestAzureCliPerf:
-    def test_pfazure_run_create(self, operation_scope_args, runtime: str, randstr: Callable[[str], str], time_limit=30):
+    def test_pfazure_run_create(self, operation_scope_args, runtime: str, randstr: Callable[[str], str], time_limit=15):
```

Comment:
nit: can we move `time_limit` into inner place, for example, inside `mock_log_activity`? seems we just pass it in to that function.

Reply:
also, as all `time_limit` is 15 now, is it possible to make it a constant?

Reply:
I don't think it's possible. We cannot require that the time limit for all tests be consistent. Although I have manually changed it to be consistent now, we will continue to modify it according to the actual situation in the future.

Reply:
"**time_limit**" is not used in "**mock_log_activity**", it is for the duration of each test, It is used in the **run_command** function.